src/guide/transitions-list.md

```diff
@@ -0,0 +1,439 @@
+
+These FLIP animations are also not limited to a single axis. Items in a multidimensional grid can be [transitioned too](https://codesandbox.io/s/github/vuejs/vuejs.org/tree/master/src/v2/examples/vue-20-list-move-transitions):
+
+TODO: example
+
+### Staggering List Transitions
+
+By communicating with JavaScript transitions through data attributes, it's also possible to stagger transitions in a list:
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.3.4/gsap.min.js"></script>
+
+<div id="demo">
+  <input v-model="query" />
+  <transition-group
+    name="staggered-fade"
+    tag="ul"
+    :css="false"
+    @before-enter="beforeEnter"
+    @enter="enter"
+    @leave="leave"
+  >
+    <li
+      v-for="(item, index) in computedList"
+      :key="item.msg"
+      :data-index="index"
+    >
+      {{ item.msg }}
+    </li>
+  </transition-group>
+</div>
+```
+
+```js
+const Demo = {
+  data() {
+    return {
+      query: '',
+      list: [
+        { msg: 'Bruce Lee' },
+        { msg: 'Jackie Chan' },
+        { msg: 'Chuck Norris' },
+        { msg: 'Jet Li' },
+        { msg: 'Kung Fury' }
+      ]
+    }
+  },
+  computed: {
+    computedList() {
+      var vm = this
+      return this.list.filter(item => {
+        return item.msg.toLowerCase().indexOf(vm.query.toLowerCase()) !== -1
+      })
+    }
+  },
+  methods: {
+    beforeEnter(el) {
+      el.style.opacity = 0
+      el.style.height = 0
+    },
+    enter(el, done) {
+      gsap.to(el, {
+        opacity: 1,
+        height: '1.6em',
+        delay: el.dataset.index * 0.15,
+        onComplete: done
+      })
+    },
+    leave(el, done) {
+      gsap.to(el, {
+        opacity: 0,
+        height: 0,
+        delay: el.dataset.index * 0.15,
+        onComplete: done
+      })
+    }
+  }
+}
+
+Vue.createApp(Demo).mount('#demo')
+```
+
+<p class="codepen" data-height="300" data-theme-id="39028" data-default-tab="js,result" data-user="Vue" data-slug-hash="c2fc5107bd3025ceadea049b3ee44ec0" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="Staggered Lists">
+  <span>See the Pen <a href="https://codepen.io/team/Vue/pen/c2fc5107bd3025ceadea049b3ee44ec0">
+  Staggered Lists</a> by Vue (<a href="https://codepen.io/Vue">@Vue</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://static.codepen.io/assets/embed/ei.js"></script>
+
+## Reusable Transitions
+
+Transitions can be reused through Vue's component system. To create a reusable transition, all you have to do is place a `<transition>` or `<transition-group>` component at the root, then pass any children into the transition component.
+
+Here's an example using a template component:
+
+```js
+Vue.component('my-special-transition', {
```

Comment:
**chore**: these two examples should be refactored to Vue 3 as well, worth a TODO 😅

Reply:
Added! :)